src/components/List.js

```diff
@@ -125,6 +128,13 @@ const List = ({ token }) => {
     if (daysUntilNextPurchase >= 30) return 'not-too-soon';
   };
 
+  const checkboxColor = {
+    inactive: 'lightgrey',
+    soon: 'red',
+    'kind-of-soon': 'orange',
```

Comment:
I think you need the quotes on the keys that have hyphens like this, but I could be wrong. You won't need them on the others, for sure.

Reply:
Oh ok, so do you think prettier changed it?  Because I initially had quotes on them all, then noticed it was like this.

Reply:
Yeah...this is correct. 👍